systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -174,6 +184,162 @@ void testJSONFormatLogging() {
         assertThat(StUtils.checkLogForJSONFormat(eoPods, "user-operator"), is(true));
     }
 
+    @Test
+    @Order(14)
+    @SuppressWarnings({"checkstyle:MethodLength"})
+    void testDynamicallySetEOloggingLevels() throws InterruptedException {
```

Comment:
The test will fail, because Kafka is not setup. Can you add it there? Or should I take a look at it?